metatensor-torch/include/metatensor/torch/atomistic/model.hpp

```diff
@@ -274,6 +276,9 @@ class METATENSOR_TORCH_EXPORT ModelMetadataHolder: public torch::CustomClassHold
     /// - "model": for reference specific to this exact model
     torch::Dict<std::string, std::vector<std::string>> references;
 
+    /// Extra metadata about this model. This can be anything.
+    torch::Dict<std::string, std::string> extra_metadata;
```

Comment:
you will also need to add this to `ModelMetadata` serialization

Reply:
Nitpick: should this be named `extra`? Since it will be accessed as `metadata.extra["something"]`, which feels nicer than `metadata.extra_metadata["something"]`

Reply:
Whoops, I forgot. And yes, changed to `extra`